src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,57 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private static final int MAX_VALUE = 11;
+    private static final int DEFAULT_INDEX = -1;
```

Comment:
Wrong name DEFAULT and INDEX concern different aspects (e.g. DEFAULT_VALUE or INITIAL_INDEX)